src/domain/permissions.js

```diff
@@ -1,70 +1,35 @@
 import { ROLE } from './constants.js';
 
-const ADMIN = 'admin';
-const EDIT_DOC = 'edit-doc';
-const VIEW_DOCS = 'view-docs';
-const EDIT_FILE = 'edit-file';
-const OWN_ROOMS = 'own-rooms';
-const REVIEW_DOC = 'review-doc';
-const VERIFY_DOC = 'verify-doc';
-const VIEW_FILES = 'view-files';
-const JOIN_ROOMS = 'join-rooms';
-const CREATE_FILE = 'create-file';
-const ARCHIVE_DOC = 'archive-doc';
-const MIGRATE_DATA = 'migrate-data';
-const SEARCH_USERS = 'search-users';
+const VIEW_USERS = 'view-users';
 const MANAGE_USERS = 'manage-users';
-const MANAGE_BATCHES = 'manage-batches';
-const SEE_USER_EMAIL = 'see-user-email';
-const MANAGE_CONTENT = 'manage-content';
-const PROTECT_ANY_DOC = 'protect-any-doc';
-const PROTECT_OWN_DOC = 'protect-own-doc';
-const MANAGE_SETTINGS = 'manage-settings';
-const DELETE_OWN_FILES = 'delete-own-files';
-const HARD_DELETE_SECTION = 'hard-delete-section';
-const DELETE_FOREIGN_ROOMS = 'delete-foreign-rooms';
-const MANAGE_STORAGE_PLANS = 'manage-storage-plans';
-const DELETE_ANY_STORAGE_FILE = 'delete-storage-file';
-const RESTORE_DOC_REVISIONS = 'restore-doc-revisions';
-const CREATE_DOCUMENT_COMMENTS = 'create-document-comments';
-const MANAGE_DOCUMENT_COMMENTS = 'manage-document-comments';
-const AUTORIZE_ROOMS_RESOURCES = 'authorize-room-resources';
-const MANAGE_ACCREDITED_EDITORS = 'manage-accredited-editors';
+const MANAGE_SETUP = 'manage-setup';
+const CREATE_CONTENT = 'create-content';
+const ACCESS_STORAGE = 'access-storage';
+const BATCH_PROCESS_DATA = 'batch-process-data';
+const MANAGE_PUBLIC_CONTENT = 'manage-public-content';
+const PROTECT_OWN_PUBLIC_CONTENT = 'protect-own-public-content';
+const DELETE_OWN_PRIVATE_CONTENT = 'delete-own-private-content';
+const DELETE_ANY_PRIVATE_CONTENT = 'delete-any-private-content';
+const MANAGE_PROTECTORS_OF_OWN_PUBLIC_CONTENT = 'manage-protectors-of-own-public-content';
 
 const userPermissions = [
-  EDIT_DOC,
-  VIEW_DOCS,
-  EDIT_FILE,
-  VIEW_FILES,
-  DELETE_OWN_FILES,
-  CREATE_FILE,
-  OWN_ROOMS,
-  AUTORIZE_ROOMS_RESOURCES,
-  JOIN_ROOMS,
-  CREATE_DOCUMENT_COMMENTS
+  CREATE_CONTENT,
+  ACCESS_STORAGE,
+  DELETE_OWN_PRIVATE_CONTENT
 ];
 
 const accreditedAuthorPermissions = [
   ...userPermissions,
-  PROTECT_OWN_DOC
+  PROTECT_OWN_PUBLIC_CONTENT
 ];
 
 const maintainerPermissions = [
   ...new Set([
     ...userPermissions,
     ...accreditedAuthorPermissions,
-    HARD_DELETE_SECTION,
-    DELETE_ANY_STORAGE_FILE,
-    SEARCH_USERS,
-    SEE_USER_EMAIL,
-    RESTORE_DOC_REVISIONS,
-    ARCHIVE_DOC,
-    REVIEW_DOC,
-    VERIFY_DOC,
-    PROTECT_ANY_DOC,
-    MANAGE_DOCUMENT_COMMENTS,
-    MANAGE_CONTENT,
-    MANAGE_ACCREDITED_EDITORS
+    VIEW_USERS,
+    MANAGE_PUBLIC_CONTENT,
+    MANAGE_PROTECTORS_OF_OWN_PUBLIC_CONTENT
```

Comment:
Honestly, curating the list of accredited authors on a document seems to me very much in the same category as setting the protected flag or writing review text etc. In other words, I would also get rid of this fine-grained permission and just let users with `MANAGE_PUBLIC_CONTENT` (un)assign the accreited authors. That would also solve the conundrum that the users in this list do not necessarily have to have `PROTECT_OWN_PUBLIC_CONTENT` permissions. A maintainer can add any old user in there.